app/javascript/components/async-credentials/edit-password-field.jsx

```diff
@@ -41,6 +41,7 @@ const EditPasswordField = ({ FieldProvider, ...props }) => (
             {...rest}
             disabled={editMode || isDisabled}
             type="password"
+            autocomplete="off"
```

Comment:
I gues here we would need the `new-password` because of the weird behavior in DDF that prefills the password fields in credential forms and makes them hard to click:
![Screenshot from 2020-06-12 08-48-28](https://user-images.githubusercontent.com/649130/84473525-84661100-ac89-11ea-9749-1d56162acb07.png)

Reply:
This would make https://github.com/ManageIQ/manageiq-providers-openstack/pull/604 global ans obsolete it.